src/InlineFilesTransformer.ts

```diff
@@ -52,24 +56,11 @@ const REQUIRE = 'require';
  */
 const TRANSFORM_PROPS = [TEMPLATE_URL, STYLE_URLS];
 
-/**
- * Transformer ID
- * @internal
- */
-export const name = 'angular-component-inline-files';
-
-// increment this each time the code is modified
-/**
- * Transformer Version
- * @internal
- */
-export const version = 1;
-
 /**
  * The factory of hoisting transformer factory
  * @internal
  */
-export function factory(cs: ConfigSet) {
+export function factory(cs: ConfigSet & { compilerModule: TTypeScript }) {
```

Comment:
This factory type isn’t the same as the one in AngularConstructorParametersTransformer.ts. Is it intended ?